demonstrations_v2/tutorial_lattice_surgery/demo.py

```diff
@@ -0,0 +1,342 @@
+because we know this enables universal quantum computing, as illustrated in, e.g., the :doc:`Game of Surface Codes <demos/tutorial_game_of_surface_codes>` [#Litinski]_).
+The gist of it is that `Pauli product rotations <https://pennylane.ai/compilation/pauli-product-rotations>`__ (PPRs) like :math:`e^{-i \tfrac{\theta}{2} P}`, represented as
+
+.. figure:: ../_static/demonstration_assets/lattice_surgery/PPR.png
+    :align: center
+    :width: 50%
+    :target: javascript:void(0)
+
+for arbitrary :class:`Pauli words <~.pauli.PauliWord>` :math:`P`
+with Clifford and non-Clifford angles :math:`\theta \in \{k \tfrac{\pi}{4} | k \in \mathbb{Z} \}` cover the `(Clifford + T) <https://pennylane.ai/compilation/clifford-t-gate-set>`__ gate set, and thus allow for universal quantum computing.
+These PPRs can be directly executed using PPMs in the following way, so all we need to show is that the lattice surgery based quantum computer can perform arbitrary PPMs. Typically, this involves an auxiliary qubit in a specific state.
+Clifford angles that are odd multiples of :math:`\frac{\pi}{2}` can be performed with an auxiliary qubit in :math:`|0\rangle` like so:
+
+.. figure:: ../_static/demonstration_assets/lattice_surgery/clifford_PPM.png
+    :align: center
+    :width: 50%
+    :target: javascript:void(0)
+
+We use the color coding from [#Litinski]_, where orange boxes correspond to Clifford gates, 
+gray boxes to Pauli operators (as :math:`e^{-i \tfrac{(2k+1)\pi}{2}P} \propto P`), green boxes to non-Clifford gates, and
+blue boxes to measurements.
+Non-clifford PPRs can be realized using a magic resource state in the following way:
+
+.. figure:: ../_static/demonstration_assets/lattice_surgery/non_clifford_PPM.png
+    :align: center
+    :width: 50%
+    :target: javascript:void(0)
+
+Here, we injected a magic state :math:`T|+\rangle = \tfrac{1}{\sqrt{2}} \left(|0\rangle + e^{-i \tfrac{1}{4}} |1\rangle\right)` that was produced separately via `magic state distillation <https://pennylane.ai/qml/demos/tutorial_magic_state_distillation>`__.
+Note that Pauli operations that have angles that are multiples of 
+:math:`\pi` do not need to be executed on hardware, but can be tracked in software.
+Further, we stress that there are different circuit identities to realize PPRs via PPMs, with the ones shown here just the basic examples taken from [#Litinski]_.
+For our purposes here it suffices to note that realizing PPRs via PPMs is possible, so the only thing left to show is how to perform arbitrary PPMs using lattice surgery.
+
+Single qubit measurements
+-------------------------
+
+Before we get to the meat of performing arbitrary PPMs with lattice surgery, let us recal how to perform single qubit measurements (and thus single qubit gates).
+In the orientation we previously used, logical operators :math:`Z_L` and :math:`X_L` of the logical qubit are defined by a horizontal line of :math:`Z` and vertical line of :math:`X` measurements
```

Comment:
This feels better to me personally, both seem to be correct.
```suggestion
In the orientation we used previously, logical operators :math:`Z_L` and :math:`X_L` of the logical qubit are defined by a horizontal line of :math:`Z` and vertical line of :math:`X` measurements
```